codelab-master/apps/playground/src/app/quiz/containers/quiz/quiz.component.html

```diff
@@ -0,0 +1,121 @@
+      <!-- the scoreboard area -->
+      <codelab-scoreboard></codelab-scoreboard>
+
+      <!-- the question/explanation area -->
+      <section id="question" [ngClass]="!isAnswered() ? 'unanswered': 'answered'">
+        <span *ngIf="!isAnswered()" i18n>{{ question?.questionText }}&nbsp;
+          <span class="number-correct" *ngIf="numberOfCorrectAnswers > 1">
+            <em>({{ numberOfCorrectAnswers }} answers are correct.)</em>
+          </span>
+        </span>
+        <span *ngIf="isAnswered()" i18n>
+          {{ numberOfCorrectAnswers === 1 ?
+            'Option ' + correctOptions + ' was correct because ' :
+            'Options ' + correctOptions + ' were correct because ' }}
+          {{ explanationText }}.
+        </span>
+      </section>
+
+      <!-- the options -->
+      <codelab-quiz-question
+        [question]="question"
+        (answer)="selectedAnswer($event)">
+      </codelab-quiz-question>
+    </mat-card-content>
+
+    <mat-card-footer>
+      <!-- selection message -->
+      <span *ngIf="!isAnswered()" class="selection-message">
+        Please select an option to continue&hellip;
+      </span>
+
+      <!-- paging -->
+      <section class="paging">
+        <mat-card-actions>
+          <mat-nav-list>
+            <div class="prev-question-nav" *ngIf="question && questionIndex > 1">
+              <button type="button"
+                      mat-flat-button
+                      (click)="advanceToPreviousQuestion()"
+                      (window:keydown.ArrowLeft)="advanceToPreviousQuestion()"
+                      matTooltip="&laquo; Previous Question"
+                      matTooltipPosition="above">
+                <svg viewBox="0 0 410.258 410.258">
+                  <polygon [attr.points]="paging.previousButtonPoints"/>
+                </svg>
+              </button>
+            </div>
+
+            <div class="restart-nav" *ngIf="question && questionIndex > 1 && questionIndex < totalQuestions">
+              <button type="button"
+                      mat-flat-button
+                      (click)="restartQuiz()"
+                      matTooltip="Restart Quiz"
+                      matTooltipPosition="above">
+                <svg viewBox="0 0 305.836 305.836">
+                  <path [attr.d]="paging.restartButtonPath"/>
+                </svg>
+              </button>
+            </div>
+
+            <div class="next-question-nav" *ngIf="question && questionIndex !== totalQuestions"
+                 matTooltip="Next Question &raquo;" matTooltipPosition="above" [matTooltipDisabled]="false">
+              <button type="button"
+                      mat-flat-button
+                      [disabled]="!isAnswered()"
+                      (click)="advanceToNextQuestion()"
+                      (window:keydown.ArrowRight)="advanceToNextQuestion()"
+                      (window:keydown.Enter)="advanceToNextQuestion()">
+                <svg viewBox="0 0 410.258 410.258">
```

Comment:
can we just use material icons here?

Reply:
Ok, added the icons, but the buttons are inactive due to separating form into components.

Reply:
will work on navigation